pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -106,13 +111,40 @@ let
   wxGTK = wxGTK32;
   python = python3;
   wxPython = python.pkgs.wxPython_4_2;
+  thirdPartyPackagePath = "package.zip";
+  thirdPartyPackagesDrvs = map (pkg: pkg.override { inherit thirdPartyPackagePath python3; }) thirdPartyPackages;
+
+  thirdPartyPackagesJoined =
+    runCommand "thirdPartyPackagesJoined"
+      {
+        inherit thirdPartyPackagesDrvs;
+        nativeBuildInputs = [unzip jq];
```

Comment:
minor style point, but you seem to otherwise be consistent with it:
single line lists are usually padded with spaces inside the square brackets

Reply:
Done!